### DISCUSSION TITLE
unable to see the floating toolbar

### DISCUSSION THREAD

**Discussion by hali20** [pubpub-zz/ppInk#318] [Q&A] [answered]
I've just installed PPInk and PPInkS both but none of them is showing floating toolbar, I checked options and the box is checked for that. I'm having Windows 11
any suggestion on what am i doing wrong? is there anything I need to enable ? please help

**Comment by hali20** [comment, marked as answer]
i was able to resolve this by reading the discussion **missing pop-out toolbar when double monitor #257**
here https://github.com/pubpub-zz/ppInk/discussions/257

**Comment by pubpub-zz** [comment]
do not hesitate to star the project 😉
